src/Tools/BuildBoss/Program.cs

```diff
@@ -0,0 +1,94 @@
+﻿using Newtonsoft.Json;
```

Comment:
Nit: Generally we have the "Place System directives first" option checked for Roslyn development.